CalcualtorLogic.js

```diff
@@ -160,13 +160,13 @@ class Calculator {
             let firstOperand = parseFloat(this.lastValue);
             let secondOperand = parseFloat(this.currentResult);
             let _result;
-            if (this.operator === "+") _result = calculatorLogic.add(firstOperand, secondOperand);
-            if (this.operator === "-") _result = calculatorLogic.subtract(firstOperand, secondOperand);
-            if (this.operator === "*") _result = calculatorLogic.multiply(firstOperand, secondOperand);
-            if (this.operator === "/") _result = calculatorLogic.divide(firstOperand, secondOperand);
+            if (this.operator === "+") _result = this.calculatorLogic.add(firstOperand, secondOperand);
+            if (this.operator === "-") _result = this.calculatorLogic.subtract(firstOperand, secondOperand);
+            if (this.operator === "*") _result = this.calculatorLogic.multiply(firstOperand, secondOperand);
+            if (this.operator === "/") _result = this.calculatorLogic.divide(firstOperand, secondOperand);
             this.currentResult = _result;
-            if (this.operator === "/" && secondOperand === 0);
-            this.currentResult = 'Cannot divide by zero';
+            if (this.operator === "/" && secondOperand === 0)
+                this.currentResult = 'Cannot divide by zero';
```

Comment:
this code 

```js 

if (this.operator === "/" && secondOperand === 0)
                this.currentResult = 'Cannot divide by zero';

```
need to be handled/moved to the `CalculatorLogic` 
this responsibility of the `CalculatorLogic` 

Please I don't you expect you to move the code without thinking 

you need to refactor the `divide` method in the `CalculatorLogic` class to move the code above.

in other words, the `divide` method needs to return the result 
now the result might be error message or a value